### ISSUE TITLE
ask the optimizer and learning rate attenuation strategy about VQvae

### ISSUE THREAD

**Issue by GANG370** [kai-wen-yang/CD-VAE#7]
When you use VQ-vae for high-precision picture training, how do you set the optimizer and learning rate? I did not get good results when I used the optimizer and learning rate attenuation strategy in your original paper. Please help me, thank you very much

**Comment by kai-wen-yang** [comment]
I use the same optimizer and learning rate scheduler, i.e., AdamW and ConsineAnnealingLR.

**Comment by GANG370** [comment]
Your program: scheduler = torch. Optim. Lr_scheduler. CosineAnnealingLR (optimizer, T_max = **50**, eta_min = learning_rate_min)
 

In the paper, the epoch for high-precision pictures is set to **60.** Is the T_max parameter still 50? Because my task training time is too long, I want to confirm the specific parameters, thank you for your reply
